### PR TITLE
Remove unnecessary abs() calls in scale/opacity loss calculation

### PR DESCRIPTION
**Pull request by vincentwoo**
the outputs of torch.sigmoid and torch.exp are always positive, therefore we need not call torch.abs on their results.

### CODE REVIEW

**Review by liruilong940607 — APPROVED** [review]
LGTM
